docs/Installing.rst

```diff
@@ -28,14 +28,9 @@ both DF and DFHack.
 Windows
 -------
 
-* DFHack only supports the SDL version of Dwarf Fortress. The "legacy" version
-  will *not* work with DFHack (the "small" SDL version is acceptable, however).
-* Windows XP and older are *not* supported, due in part to a
-  `Visual C++ 2015 bug <https://stackoverflow.com/questions/32452777/visual-c-2015-express-stat-not-working-on-windows-xp>`_
-
-The Windows build of DFHack should work under Wine on other operating systems,
-although this is not tested very often. It is recommended to use the native
-build for your operating system instead.
+The Windows build of DFHack should work under Wine on other operating systems.
```

Comment:
I think we should keep the XP note. Feels weird to have the only text under "Windows" be about other systems.

Reply:
done